google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/models/BulkMutationTest.java

```diff
@@ -0,0 +1,119 @@
+
+  @Test
+  public void test() {
+    BulkMutation m =
+        BulkMutation.create(TABLE_ID)
+            .add(
+                "key-a",
+                Mutation.create()
+                    .setCell("fake-family1", "fake-qualifier1", 1_000, "fake-value1")
+                    .setCell("fake-family2", "fake-qualifier2", 2_000, "fake-value2"))
+            .add(
+                ByteString.copyFromUtf8("key-b"),
+                Mutation.create().setCell("fake-family3", "fake-qualifier3", 3_000, "fake-value3"));
+
+    MutateRowsRequest actual = m.toProto(REQUEST_CONTEXT);
+    assertThat(actual)
+        .isEqualTo(
+            MutateRowsRequest.newBuilder()
+                .setTableName(
+                    TableName.format(
+                        INSTANCE_NAME.getProject(), INSTANCE_NAME.getInstance(), TABLE_ID))
+                .setAppProfileId(APP_PROFILE)
+                .addEntries(
+                    Entry.newBuilder()
+                        .setRowKey(ByteString.copyFromUtf8("key-a"))
+                        .addMutations(
+                            com.google.bigtable.v2.Mutation.newBuilder()
+                                .setSetCell(
+                                    SetCell.newBuilder()
+                                        .setFamilyName("fake-family1")
+                                        .setColumnQualifier(
+                                            ByteString.copyFromUtf8("fake-qualifier1"))
+                                        .setTimestampMicros(1_000)
+                                        .setValue(ByteString.copyFromUtf8("fake-value1"))))
+                        .addMutations(
+                            com.google.bigtable.v2.Mutation.newBuilder()
+                                .setSetCell(
+                                    SetCell.newBuilder()
+                                        .setFamilyName("fake-family2")
+                                        .setColumnQualifier(
+                                            ByteString.copyFromUtf8("fake-qualifier2"))
+                                        .setTimestampMicros(2_000)
+                                        .setValue(ByteString.copyFromUtf8("fake-value2")))))
+                .addEntries(
+                    Entry.newBuilder()
+                        .setRowKey(ByteString.copyFromUtf8("key-b"))
+                        .addMutations(
+                            com.google.bigtable.v2.Mutation.newBuilder()
+                                .setSetCell(
+                                    SetCell.newBuilder()
+                                        .setFamilyName("fake-family3")
+                                        .setColumnQualifier(
+                                            ByteString.copyFromUtf8("fake-qualifier3"))
+                                        .setTimestampMicros(3_000)
+                                        .setValue(ByteString.copyFromUtf8("fake-value3")))))
```

Comment:
This is a *lot* of indentation. Could you break this up into some temporary variables?

Reply:
It's still pretty ugly with temp vars. I switched it to a text proto